cmdline-pull/pubsub_sample.py

```diff
@@ -42,7 +42,7 @@
   PROJ delete_subscription SUBSCRIPTION
   PROJ connect_irc TOPIC SERVER CHANNEL
   PROJ publish_message TOPIC MESSAGE
-  PROJ pull_messages SUBSCRIPTION
+  PROJ pull_messages SUBSCRIPTION [LOOP=True|False (Default: True)]
```

Comment:
It is bit confusing that whether we should do the following
```
$ pubsub-sample.py proj pull_messages my-subscription LOOP=True
```
or
```
$ pubsub-sample.py proj pull_messages my-subscription True
```

How about
```
PROJ pull_messages SUBSCRIPTION [LOOP](True|False Default: True)
```
or use the argparser?